### PR TITLE
Add documentation for redfish alert  enabling for Dell nodes

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
@RackHD/veyron @RackHD/corecommitters

File: docs/rackhd/event_notification.rst [hunk 1]
```diff
@@ -420,6 +420,68 @@ If the endpoint is redfish enabled and supports the Resfish EventService, it is
...
+
+.. code-block:: REST
+
+    {
+      "@odata.context": "/redfish/v1/$metadata#EventDestination.EventDestination",
+      "@odata.id": "/redfish/v1/EventService/Subscriptions/b50106d4-32c6-11e7-8b05-64006ac35232",
+      "@odata.type": "#EventDestination.v1_0_2.EventDestination",
+      "Context": "RackhHD Subscription",
+      "Description": "Event Subscription Details",
+      "Destination": "https://10.240.19.226:8443/api/2.0/notification/alerts",
```
Comment (brianparry) [inline docs/rackhd/event_notification.rst]: @tannoa2 You make this a generic IP

File: docs/rackhd/event_notification.rst [hunk 2]
```diff
@@ -420,6 +420,68 @@ If the endpoint is redfish enabled and supports the Resfish EventService, it is
...
+If the node is a Dell node, it is possible to post the Graph.Dell.Configure.Redfish.Alerting workflow.
+The workflow will:
+
+1- Enable Alerts for the Dell node. Equivalent to running "set iDRAC.IPMILan.AlertEnable 1" racadam command.
+
+2- Enable redfish alerts. Equivalent to running "eventfilters set -c idrac.alert.all -a none -n redfish-events" racadam command.
+
+3- Disable the "Audit" info alerts. Equivalent to running  "eventfilters set -c idrac.alert.audit.info -a none -n none" racadam command.
+
+The workflow will run the default values if the node's obm is set and the "rackhdPublicId" property is set in the rackHD config.json file.
```
Comment (anhou) [inline docs/rackhd/event_notification.rst]: rackhdPublicId  should be 'rackhdPublicIp' according to https://github.com/RackHD/RackHD/pull/741

**Comment by tannoa2** [comment]
test this please

**Pull request merged**